File: mobile-development/src/main/java/org/usi/lesson_2020_11_06/Collections.java
package org.usi.lesson_2020_11_06;

import java.util.ArrayList;
import java.util.List;

public class Collections {

    public static void main(String[] args) {
        List<String> list = new ArrayList<>();
        list.add("item 1");
        list.add("item 2");
        list.add("item 3");
        list.add("item 4");

        List<String> list2 = List.of("1", "2");

        for (String item : list) {
            System.out.println(item);
        }
    }

}

File: mobile-development/src/main/kotlin/org/uis/Collections.kt
package org.uis

import java.util.ArrayList
import kotlin.jvm.JvmStatic

object Collections {

    @JvmStatic
    fun main(args: Array<String>) {

        val list = mutableListOf("one", "two", "three", "four")

        for (item in list) {
            println(item)
        }

    }
}